bound/typescript/tests/vectors/http/exchanges/create-exchange.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
@nitro-neal check it, `vectors/` here

I generated these values from our kt example app, and checked them in here. They're here right now for sanity, so we may choose to move these to official vectors in due time, or maybe find a different testing solution altogether. Just making you aware.

Reply:
yup awesome, will add a task in our running list to add these to official test vectors basically

Reply:
probably have a new folder for end2end flow or something, maybe doesn't belong in official test vector spot but good to have